lib/vrl/parser/src/ast.rs

```diff
@@ -806,7 +850,7 @@ impl fmt::Debug for AssignmentTarget {
 // query
 // -----------------------------------------------------------------------------
 
-#[derive(PartialEq)]
+#[derive(Clone, PartialEq)]
```

Comment:
The added `Clone`s is unfortunate.

As discussed on Discord with @FungusHumungus, it should be possible to remove these, but we'll defer that until after 0.12 is released. Could you create a tracking issue for this @FungusHumungus?